### ISSUE TITLE
What is the role of 'reset_running_statistics' ?

### ISSUE THREAD

**Issue by zanonShao** [mit-han-lab/once-for-all#15]
On line 67 of **progressive_shrinking.py**, why do we need the _'reset_running_statistics'_ function to reset both the 'mean' and 'var' value of the batchnormal layer to the 'mean' and 'var' obtained from random 2000 images?
`run_manager.reset_running_statistics(dynamic_net)`

**Comment by bhpfelix** [comment]
It looks like a batch norm calibration procedure for the subnet. The **Post-Statistics of Batch Normalization** section of [this paper](https://arxiv.org/pdf/1903.05134.pdf) seems relevant.

**Comment by zanonShao** [comment]
> It looks like a batch norm calibration procedure for the subnet. The **Post-Statistics of Batch Normalization** section of [this paper](https://arxiv.org/pdf/1903.05134.pdf) seems relevant.

You are right, that's very kind of you!  : )
